src/third_party/histogram/histogram.hpp

```diff
@@ -75,13 +75,15 @@ class Histogram
   {
     if( x < Start )
       ++underflow;
+    else if( x > End )
+      ++overflow;
     else
     {
       const size_t i(
         static_cast<size_t>(
         (x-Start)*nBins_by_interval) );
-      if( i < nBins ) ++freq[i];
-      else ++overflow;
+      // clamp for the particular case when (x == End)
```

Comment:
A test has been added line 78.
This way we can deal with the cases when x == End